mangaki/mangaki/models.py

```diff
@@ -55,6 +235,15 @@ def search(self, search_text):
         return self.filter(title__search=search_text).\
             order_by(SearchSimilarity(F('title'), Value(search_text)).desc())
 
+    def dpp(self,category, nb_points):
+
+        build_matrix = RatingsMatrix()
+        matrix = build_matrix.build_matrix()
+        similarity = SimilarityMatrix(matrix, nb_components_svd=70)
+        items = list(build_matrix.item_dict.values())
```

Comment:
Pardon, `items = [build_matrix.item_dict[item] for item in self.values_list('pk')]`